app/script/calling/entities/CallEntity.js

```diff
@@ -135,7 +133,11 @@ z.calling.entities.CallEntity = class CallEntity {
       return false;
     });
 
-    this.participantsCount = ko.pureComputed(() => this.getNumberOfParticipants(this.selfUserJoined()));
+    this.participants.subscribe(participants => {
+      const additionalCount = this.selfClientJoined() ? 1 : 0;
+      const numberOfParticipants = participants.length + additionalCount;
+      this.telemetry.numberOfParticipantsChanged(numberOfParticipants);
+    });
```

Comment:
Doesn't work incase 4 people have joined the call and then you join as the last person. Max participants should be 5 but it will remain 4 as this does not trigger again.